### DISCUSSION TITLE
Configure proportional resizing in WYSIWYG

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12646] [Feature Requests]
Following [TinyMCE / Advanced editing behaviors](https://www.tiny.cloud/docs/configure/advanced-editing-behavior/), I tried 
```
{
    "object_resizing": "img",
    "resize_img_proportional": "true"
}
```
in the field `Options Override` of the WYSIWYG, but it does not work

The goal is to have the user to set one dimension, the `Width` or the `Height` and to have the other dimensin automatically set proportionally.

**Comment by rijkvanzanten** [comment]
I don't think there's currently a configuration that makes this possible. Lets treat this discussion as the feature request for such a feature 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
